src/transformStateWithClones.js

```diff
@@ -7,7 +7,39 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const res = [];
+  let newState = {
+    ...state,
+  };
+  const addProp = 'addProperties';
+  const removeProp = 'removeProperties';
+  const clear = 'clear';
```

Comment:
It's cool that you decided to put the type names into separate constants. 🔥 
But then they should be taken out of the function definition altogether (so that they are not created anew every time when the function is called) and called in uppercase style.